src/main/java/org/trigon/utilities/mirrorable/Mirrorable.java

```diff
@@ -40,7 +40,7 @@ public static boolean isRedAlliance() {
     }
 
     /**
-     * Gets a command that updates the alliance. This is used to cache the alliance every 0.5 seconds. Ignoring disable is used to allow this command to run when the robot is disabled.
+     * Gets a command that updates the alliance. This is used to cache the alliance every 0.5 seconds. Ignoring disable is used to update the alliance when the robot is disabled.
```

Comment:
Current alliance, or really anything that explains clearer that it doesn't "update the alliance"